src/pages/index.module.css

```diff
@@ -1,23 +1,394 @@
-/**
- * CSS files with the .module.css suffix will be treated as CSS modules
- * and scoped locally.
- */
+/* ----- Elements ------  */
 
-.heroBanner {
-	padding: 4rem 0;
+:root {
+	--pink: #f37;
+	--blue: #2563eb;
+	--bg-blue: #e0f2fe;
+}
+
+.main code {
+	font-size: 0.85rem !important;
+}
+
+.main {
+	margin: 1rem;
+}
+
+.main > section {
+	max-width: 50rem;
+	margin: 0 auto;
+	margin-top: 5rem;
+}
+
+.main section h2,
+.main section p {
+	max-width: 30rem;
+}
+
+.button {
+	color: var(--blue);
+	font-weight: 600;
+	background-color: var(--bg-blue);
+	padding: 0.5rem;
```

Comment:
```suggestion
	padding: 0.5rem 1rem;
```


Diff: 
![Screenshot 14-03-2023 at 15 22 03](https://user-images.githubusercontent.com/375537/225031330-91517d6b-2983-4d17-998e-edf69566fe2b.gif)